worlds/minecraft/Rules.py

```diff
@@ -261,6 +264,26 @@ def get_rules_lookup(player: int):
             "Sneak 100": lambda state: can_adventure(state, player) and has_iron_ingots(state, player) and state.has("Progressive Tools", player, 2),
             "When the Squad Hops into Town": lambda state: can_adventure(state, player) and state.has("Lead", player),
             "With Our Powers Combined!": lambda state: can_adventure(state, player) and state.has("Lead", player),
+            "Smells Interesting": lambda state: can_excavate(state, player),
+            "Little Sniffs": lambda state: can_excavate(state, player),
+            "Planting the Past": lambda state: can_excavate(state, player),
+            "Crafting a New Look": lambda state: (has_iron_ingots(state, player) and # Maybe streamline this one
+                (can_adventure(state, player) or
+                    fortress_loot(state, player) or
+                    (state.can_reach("Pillager Outpost", 'Region', player) and basic_combat(state, player)) or
+                    (state.can_reach("Bastion Remnant", 'Region', player) and basic_combat(state, player)) or
+                    (state.can_reach("End City", 'Region', player) and basic_combat(state, player)))),
+            "Smithing with Style": lambda state: (can_excavate(state, player) and # Wayfinder Armor Trim
+                fortress_loot(state, player) and # Rib Armor Trim
+                (state.can_reach("Village", 'Region', player) or overworld_villager(state, player)) and  # Explorer Maps for the Vex and Tide Armor Trims
+                state.can_reach("Bastion Remnant", 'Region', player) and  # Snout Armor Trim
+                state.can_reach("End City", 'Region', player) and  # Spire Armor Trim
+                state.has("Progressive Tools", player, 2) and  # Ward and Silence Armor Trims
+                ((state.has("Fishing Rod", player) and can_brew_potions(state, player)) or  # Water Breathing Potions for the Tide Armor Trim
```

Comment:
maybe alternative is having enchants for Respiration. I'm beginning to not like this advancement.

Reply:
That could be a good alternative, though Respiration is a pretty rare and specific Enchantment to have as a hard requirement in my opinion.

Reply:
Decided to add it as an additional requirement with Buckets.